nntrainer/tensor/tensor.cpp

```diff
@@ -3092,7 +3092,9 @@ void Tensor::copyData(const Tensor &from) {
       }
     } else if (getDataType() == ml::train::TensorDim::DataType::FP16) {
 #ifdef ENABLE_FP16
-      if (from.getDataType() == ml::train::TensorDim::DataType::QINT8) {
+      if (from.getDataType() == ml::train::TensorDim::DataType::FP32) {
+        scopy(size(), from.getData<float>(), 1, getData<_FP16>(), 1);
```

Comment:
I wonder how it works for the unit16 case for (_FP16). In the end, we plan to make enable_fp16 true always and it also works for unit16 case as well.

Reply:
Adding function that works like : uint16 -> __fp16 -> fp32 would do for that.
But there is no such case for now maybe?

Reply:
yes. not now. Thanks.